tests/test_unit/test_cdf_tk/test_commands/test_purge.py

```diff
@@ -105,344 +93,130 @@ def files_by_node_id(
 
 
 @pytest.fixture()
-def purge_ts_client(
-    cognite_timeseries_2000_list: NodeList[CogniteTimeSeries], timeseries_by_node_id: dict[NodeId, ExtendedTimeSeries]
-) -> Iterator[ToolkitClient]:
-    timeseries_by_id = {ts.id: ts for ts in timeseries_by_node_id.values()}
-
-    def retrieve_timeseries_mock(
-        ids: Sequence[int] | None = None,
-        external_ids: SequenceNotStr[str] | None = None,
-        instance_ids: Sequence[NodeId] | None = None,
-        ignore_unknown_ids: bool = False,
-    ) -> ExtendedTimeSeriesList:
-        if ids is not None or external_ids is not None:
-            raise ValueError("Unexpected call to retrieve_timeseries with ids or external_ids")
-        if instance_ids is None:
-            raise ValueError("instance_ids must be provided")
-        return ExtendedTimeSeriesList([timeseries_by_node_id[node_id] for node_id in instance_ids])
-
-    def unlink_instance_ids_mock(id: int | Sequence[int] | None = None, **_) -> ExtendedTimeSeriesList:
-        if id is None:
-            raise ValueError("id must be provided")
-        if isinstance(id, int):
-            id = [id]
-        return ExtendedTimeSeriesList([timeseries_by_id[i] for i in id])
-
-    with monkeypatch_toolkit_client() as client:
-        client.iam.token.inspect.return_value = TokenInspection(
-            "123",
-            projects=[],
-            capabilities=ProjectCapabilityList(
-                [
-                    ProjectCapability(
-                        capability=DataModelsAcl([DataModelsAcl.Action.Read], DataModelsAcl.Scope.All()),
-                        project_scope=AllProjectsScope(),
-                    ),
-                    ProjectCapability(
-                        capability=DataModelInstancesAcl(
-                            [DataModelInstancesAcl.Action.Read, DataModelInstancesAcl.Action.Write],
-                            DataModelInstancesAcl.Scope.All(),
-                        ),
-                        project_scope=AllProjectsScope(),
-                    ),
-                    ProjectCapability(
-                        capability=TimeSeriesAcl(
-                            [TimeSeriesAcl.Action.Read, TimeSeriesAcl.Action.Write], TimeSeriesAcl.Scope.All()
-                        ),
-                        project_scope=AllProjectsScope(),
-                    ),
-                ]
-            ),
-        )
-        client.data_modeling.views.retrieve.return_value = ViewList(
-            [
-                View(
-                    "cdf_cdm",
-                    "CogniteTimeSeries",
-                    "v1",
+def purge_responses(toolkit_config: ToolkitClientConfig) -> Iterator[responses.RequestsMock]:
+    config = toolkit_config
+    with responses.RequestsMock() as rsps:
```

Comment:
Please add a fixture for this as it would mean more readable code (you can then dedent a ton of lines)